central/debug/service/service.go

```diff
@@ -403,9 +414,16 @@ func getCentralDBData(ctx context.Context, zipWriter *zip.Writer) error {
 		return err
 	}
 
-	dbDiagnosticData := buildDBDiagnosticData(ctx, dbConfig, globaldb.GetPostgres())
-
-	return addJSONToZip(zipWriter, "central-db.json", dbDiagnosticData)
+	db := globaldb.GetPostgres()
+	dbDiagnosticData := buildDBDiagnosticData(ctx, dbConfig, db)
+	if err := addJSONToZip(zipWriter, "central-db.json", dbDiagnosticData); err != nil {
+		return err
+	}
+	statements, err := stats.GetPGStatStatements(ctx, db, pgStatStatementsMax)
```

Comment:
The extension may not be enabled for an external DB.  In such cases this would return an error.  So the question in my mind is in such cases is it sufficient to know that if the `central-db-pg-stats.json` isn't there to go look for the error in the log OR would it be more obvious to write the error to `central-db-pg-stats.json`.

Reply:
oh right! I literally was thinking about this when writing this PR then just completely forgot to add the check

Reply:
@dashrews78 I ended up embedding the error into the struct so that it'll show up in the json response. lmk what you think